### DISCUSSION TITLE
How to give execution priority to assets or ops within a run

### DISCUSSION THREAD

**Discussion by sryza** [dagster-io/dagster#14724] [Q&A] [answered]
Within a Dagster run, I want Dagster to execute some ops or assets before other ops or assets.

**Comment by sryza** [comment, marked as answer]
You can set the "dagster/priority" op tag.  The higher the number, the higher the priority. 0 is the default. Both positive and negative numbers can be used.

A higher priority will never allow a downstream asset or op to be executed before an asset or op that's upstream of it.

Asset example:

```python
@asset(op_tags={"dagster/priority": 1})
def asset1():
    ...
```
Op example:

```python
@op(tags={"dagster/priority": 1})
def op1():
    ...
```

**Reply by vladidobro** [reply]
Hi, has this feature been deprecated? I cannot find anything about it in the docs and it doesnt work for me (1.9.9)
@sryza

**Reply by garethbrickman** [reply]
@prha Is using `dagster/priority` in op tags still an available feature but just not yet in the new docs?

**Reply by prha** [reply]
@vladidobro This hasn't been deprecated, but should still work.  

I can work on adding some better docs for this.

In the meantime, if you could share a sample of what is not working for you, I can investigate.

**Reply by Rothenhouser** [reply]
Similar issue for me. I think the problem is that the op_tags aren't attached to runs started from a backfill or an AutomationCondition.
